concepts/mutability/.meta/config.json

```
@@ -1,7 +1,5 @@
 {
-    "blurb": "<todo>",
-    "authors": [
-        "misicnenad"
-    ],
-    "contributors": []
+  "blurb": "Although Cairo uses an immutable memory model by default, it offers an option to make variables appear mutable.",
+  "authors": ["okhai"],
```

```suggestion
  "authors": ["okhaimie-dev"],
```
You should use your full GH username here, see https://exercism.org/docs/building/tracks/concepts#h-file-meta-config-json